conda_recipe_manager/parser/v0_recipe_formatter.py

```
@@ -105,3 +156,16 @@ def fmt_text(self) -> None:
                 bad_lst_block_indent_tracker = -1
 
             idx += 1
+
+    def fix_excessive_indentation(self) -> bool:
+        """
+        Fixes excessive indentation in the recipe file. Excessive indentation is defined as a line that is more than 1
+        tab indented with respect to its parent line.
+
+        :returns: True if the operation was successful. False otherwise.
+        """
+        try:
+            self._fix_excessive_indentation()
+        except IndentFormattingException:
+            return False
```

It might also be worth catching a generic exception or at least index out of bounds?

---

I think that would be more useful when we have logging and can signal to the user what went wrong exactly before proceeding. Catching more generic exceptions here would allow the program to continue running after the formatting function failed silently for an unknown reason, and we wouldn't get a bug report to improve it.